packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -8,149 +8,201 @@ showTitle: true
+
+<img src="./img/compact-data-table.png"  alt="Compact table"  width="1187"/> 
 
 1. **[Toolbar](/design-guidelines/usage-and-behavior/toolbar):** Sits above the table and contains controls for manipulating table data. Common actions include filtering, sorting, and pagination.
-2. **Select all:** When present, selects all items in a table. If pagination is being used, this will only select items on the current page.
-3. **Column headers:** Should align with the content they contain. If the user is able to sort on a column, the first click on the header will sort the content of the table on the content in that column. Subsequent clicks will toggle the direction of the sort. Table data can only be sorted on one column at a time. See [Sorting by columns](#sorting-by-columns) for more information on the sort component.
+2. **[Select all](/design-guidelines/usage-and-behavior/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page.
+3. **Column headers:** Should align with the content they contain. If the user is able to sort on a column, the first click on the header will sort the content of the table on the content in that column. Subsequent clicks will toggle the direction of the sort. Table data can only be sorted on one column at a time. See [sorting by columns](#sorting-by-columns) for more information on the sort component.
 4. **Select checkbox:** Selects this row
 5. **Global actions:** Actions that apply to all selected items
 6. **Inline actions:** Actions that apply only to the current row/item
 7. **Pagination footer:** When present, provides navigation to additional pages
 
-#### When to use
-Use a standard table when the information you want to display fits into a structured, tabular format (i.e. has distinct rows and columns).
-
-#### When not to use
-Do not use for less structured or variably structured data that cannot be easily organized into columns. In these situations, use a [data list](#data-lists).
+### When to use table vs. list
 
-#### How to use
-Consider the structure of the data you want to display and organize that information into columns. Columns will typically have column headers. Every row within a table must have a consistent format. If the table row includes actions, they should always be placed in the rightmost column(s).
+**Use a table when**:
+* The information you want to display fits into a structured, tabular format (in other words, has distinct rows and columns).
 
-### Compact table
+**Don't use a table**: 
+* For less structured or variably structured data that cannot be easily organized into columns. In these situations, use a [data list](#data-lists).
 
-![compact table](./img/compact-data-table.png)
-
-#### When to use
-Use a compact table when you want to show as much data per page as possible and readability is a secondary concern.
+### Table capabilities
```

Comment:
What about an actionable table?

Reply:
The section about the actionable table has been added.